### ISSUE TITLE
Reference to `jill.py`

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaBerry/juliaberry.github.io#6]
[jill.py](https://github.com/johnnychen94/jill.py) boasts to be a "Julia Installer for Linux (and every other platform)". Python is pre-installed on raspberry pi. I have never used `jill.py`, but it seems like a very easy way of installing any version of Julia. Do anyone know if it works as intended on the RPi? Because if it does, it seems to me like the easiest way of installing the latest version of julia, and it should therefore be recomended.

**Comment by PallHaraldsson** [comment]
There's also https://github.com/JuliaLang/juliaup

>Note that the Mac and Linux version are considered prerelease, have known bugs and might often break.

The Windows version is stable, and this will likely be the future standard. Jill should also work, but (if/when the above is outdated), it seems best if one go-to solution could be recommended for all platforms.

**Comment by KronosTheLate** [comment]
Agreed. I did not know that `juliaup` had gotten to pre-release for linux - that is exciting!
However, I _still_ thing that just _mentioning_ jill is a good idea, for completeness. It is a good option (provided it works). But that the emphasis should be on `juliaup`.
